packages/components/src/components/timeline/timeline.jsx

```diff
@@ -21,7 +21,7 @@ const Timeline = ({ children, ...props }) => {
                 >
                     <Oval>{idx + 1}</Oval>
                     <div className='dc-timeline__container'>
-                        <h className='dc-timeline__title'> {child.props.title}</h>
```

Comment:
wow

Reply:
😂 , i got console error, and realize we have this in our source code